p2p/raintree/target.go

```diff
@@ -0,0 +1,45 @@
+package raintree
+
+import (
+	"fmt"
+	"strings"
+
+	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
+)
+
+type target struct {
+	Address    cryptoPocket.Address
+	ServiceUrl string
+
+	Level                  uint32
+	Percentage             float64
+	AddrBookLengthAtHeight int
+	Index                  int
+	IsSelf                 bool
+}
+
+func (t target) DebugString(r *router) string {
+	s := strings.Builder{}
+	s.WriteString("[")
+	serviceUrl := t.ServiceUrl
+	if !t.IsSelf {
+		s.WriteString(fmt.Sprintf(" (%s) ", serviceUrl))
```

Comment:
Seems like the idiomatic way of writing formatted string to a strings builder is: `fmt.Fprintf(&b, " (%s) ", serviceURL)`


Source: https://blog.boot.dev/golang/strings-builder-concatenation-golang/